From 2d36bf5177b9ec1989ecb5880936fb830196ed68 Mon Sep 17 00:00:00 2001
From: Aviana Cruz <gwencroft@proton.me>
Date: Sat, 20 Aug 2022 22:25:07 +0800
Subject: [PATCH] Fix Makefile

---
 Makefile | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 6ae2d1b..3feb6f8 100644
--- a/Makefile
+++ b/Makefile
@@ -4,15 +4,14 @@ DATA_DIR = data
 BUILD_DIR = build
 
 RM       = rm -f
-CXX      = g++
-CXXFLAGS = -std=c++11 -x c++ -Wall -fPIC
+CXXFLAGS += -std=c++11 -Wall -fPIC
 INCLUDES = -Isrc/
 LDD_DIRS =
 LDD_LIBS =
 LDD_FLAG =
 LIB_DLL  = $(BUILD_DIR)/droidcam-obs.so
 STATIC   =
 
 ifeq ($(OS),Windows_NT)
 all:
@@ -49,11 +48,8 @@ ifeq "$(ALLOW_STATIC)" "yes"
 	STATIC += $(IMOBILEDEV_DIR)/lib/libplist-2.0.a
 
 else
-	LDD_DIRS += -L$(JPEG_LIB)
-	LDD_DIRS += -L$(IMOBILEDEV_DIR)/lib
-
 	LDD_LIBS += -lturbojpeg
-	LDD_LIBS += -limobiledevice
+	LDD_LIBS += -limobiledevice-1.0
 endif
 
 INCLUDES += -I$(IMOBILEDEV_DIR)/include
